components/fieldgroup/stories/fieldgroup.test.js

```diff
@@ -19,11 +19,9 @@ export const FieldGroupSet = Variants({
 			testHeading: "Horizontal checkboxes",
 			layout: "horizontal",
 			inputType: "checkbox",
-			helpText: "Make a selection.",
```

Comment:
These aren't really necessary changes. I was thinking that we don't really need to change the default values for any of the tests, so I removed them, but that does introduce changes in the VRTs (which I ran and accepted when this PR was pointed at `main`, but haven't run since pointing to `s2-foundations-redux`).